pkg/agent/flowexporter/connections/connections_test.go

```diff
@@ -91,10 +91,12 @@ func TestConnectionStore_addAndUpdateConn(t *testing.T) {
 		TupleReply:      revTuple2,
 		IsActive:        true,
 	}
+	// To test service name realization
```

Comment:
do you mean "mapping" (as in mapping from endpoint to service name) instead of "realization" maybe?